src/utils.ts

```diff
@@ -743,23 +743,14 @@ export const parseStringParam = (
   params: URLSearchParams,
   name: string,
   defaultValue: string,
-) => {
+): string => {
   const value = params.get(name);
 
   if (value === null) {
     return defaultValue;
   }
 
-  // ?param format (no specified value)
```

Comment:
Any reason why this was dropped?

Reply:
I understand parseStringParam method must return an string, similar to parseBooleanParam and parseNumberParam which return a boolean and a number.
it was returning a number for "123" input and could return an object for a valid json object. 
Not sure if my interpretation is correct.

Reply:
Yup, that interpretation is correct and I agree with the changes after looking. Thanks!